testtools/compat.py

```diff
@@ -26,10 +25,12 @@
 import traceback
 import unicodedata
 
-from extras import try_imports
+from extras import try_import, try_imports
 
 BytesIO = try_imports(['StringIO.StringIO', 'io.BytesIO'])
 StringIO = try_imports(['StringIO.StringIO', 'io.StringIO'])
+# To let setup.py work, make this a conditional import.
```

Comment:
What does that mean exactly? What does `setup.py` do if it is not present? Do we have to do this for all of our external dependencies?